cmd/gcp-controller-manager/app/ca_cache.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/golang/glog"
+)
+
+const (
+	rootCAPrefix           = "https://pki.goog/cloud_integrity/"
+	rootCACert             = "tpm_ek_root_1.crt"
+	rootCACRL              = "tpm_ek_root_1.crl"
+	rootCAIntermediateCert = "tpm_ek_intermediate_h1_2018.crt"
```

Comment:
As the names suggest, we plan to update these every 6 months. (in fact, I guess we're overdue for the first such update).

I'd recommend instead taking the URL from the leaf cert (the authorityInfoAccess field), possibly doing some validation on it (e.g. to make sure it starts with pki.goog), and then hard-coding only the root, which should change rarely.